src/main/java/ch/njol/skript/events/EvtPeriodical.java

```diff
@@ -42,17 +42,17 @@
 	static {
 		Skript.registerEvent("*Periodical", EvtPeriodical.class, ScheduledNoWorldEvent.class, "every %timespan%")
 				.description("An event that is called periodically.")
-				.examples("every 2 seconds",
-						"every minecraft hour",
-						"every ticks #can cause lag (depends on the code in this trigger)",
-						"every minecraft days")
+				.examples("every 2 seconds:",
+						"every minecraft hour:",
+						"every tick: #can cause lag (depends on the code in this trigger)",
```

Comment:
I wrote this line myself before but now i know better
```suggestion
						"every a tick: # can cause lag depending on the code inside the event",
```